web/src/OverviewActionBar.tsx

```diff
@@ -186,6 +186,20 @@ const CustomActionButton = styled(ApiButton)`
   }
 `
 
+const DisableButton = styled(ApiButton)`
+  button {
+    ${OverviewButtonMixin};
+  }
+
+  .MuiButton-label {
+    // hardcode a width to workaround this bug:
+    // https://app.shortcut.com/windmill/story/12912/uibuttons-created-by-togglebuttons-have-different-sizes-when-toggled
+    width: ${SizeUnit(3.6)};
+  }
+
+  margin-left: ${SizeUnit(0.25)};
```

Comment:
i would recommend a larger button margin here, like maybe `0.5` or even up to `0.75` instead.

pictured below with `0.5`:
![Screen Shot 2021-11-17 at 4 56 36 PM](https://user-images.githubusercontent.com/23283986/142289144-538f3e0d-5c29-4563-9ebe-a366ec3825e6.png)

this is more a note for @SurbhiSGupta and doesn't need to be resolved in this pr! i find the placement of that button somewhat un-intuitive within the log part of the action bar, and i'm curious if others feel the same. it feels like the button gets a little lost on the far right. i tried it on both the left side and the left side with the log options centered (the latter of which may create too many layout issues on smaller screens). here are those views for reference:

![Screen Shot 2021-11-17 at 4 54 41 PM](https://user-images.githubusercontent.com/23283986/142290055-3d050d6d-f666-4137-98fc-eb5004a14e6c.png)

![Screen Shot 2021-11-17 at 4 53 57 PM](https://user-images.githubusercontent.com/23283986/142290062-8ed4fea1-f22e-4254-9b54-e3daa71d372f.png)